### PR TITLE
Remove circleci related things.

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: github-actions/bazel/configure-remote/BUILD.bazel [hunk 1]
```diff
@@ -1,7 +1,51 @@
...
+    entry_point = "index.ts",
+    format = "iife",
+    minify = True,
+    sourcemap = "",
+    deps = [":setup-bazel-remote-exec"],
+)
+
+nodejs_script_to_sh_script(
+    name = "script",
+    bundle_file = ":bundle.js",
```
Comment (devversion) [inline github-actions/bazel/configure-remote/BUILD.bazel:43]: We probably don't need this anymore?

File: ng-dev/utils/git/github.ts [hunk 2]
```diff
@@ -65,6 +66,59 @@ export class AuthenticatedGithubClient extends GithubClient {
   async graphql<T extends GraphqlQueryObject>(queryObject: T, params: RequestParameters = {}) {
     return (await this._graphql(query(queryObject).toString(), params)) as T;
   }
+
+  /**
+   * Retrieve a combined listing of the results for a refs statuses and checks.
+   */
+  async getCombinedChecksAndStatusesForRef(
+    params: RestEndpointMethodTypes['checks']['listForRef']['parameters'] &
+      RestEndpointMethodTypes['repos']['getCombinedStatusForRef']['parameters'],
```
Comment (devversion) [inline ng-dev/utils/git/github.ts:75]: Is there a way to strongly type the return type + `const results`?

File: ng-dev/utils/git/github.ts [hunk 3]
```diff
@@ -65,6 +66,59 @@ export class AuthenticatedGithubClient extends GithubClient {
...
+        name: result.context,
+        result: result.state,
+        description: result.description,
+        url: result.target_url,
+        status: result,
+      })),
+    ];
+
+    return {
+      result: results.reduce(
```
Comment (devversion) [inline ng-dev/utils/git/github.ts:99]: Nit: Probably more readable to compute in a for loop and continue early if result === failing.

**Review by devversion — APPROVED** [review]

File: ng-dev/utils/git/github.ts (hunk 3 above)
Comment (josephperrott) [reply ng-dev/utils/git/github.ts:99]: Left a TODO

File: github-actions/bazel/configure-remote/BUILD.bazel (hunk 1 above)
Comment (josephperrott) [reply github-actions/bazel/configure-remote/BUILD.bazel:43]: Left a TODO

**Comment by josephperrott** [comment]
This PR was merged into the repository by commit 2214a4943d30e2b555bf97070ccdf8358012416e.

**Pull request closed without merging**